module5/README.md

```diff
@@ -52,11 +52,11 @@ center, center
 
 n) A stack always gets one ________________ from each of its children.
 
-alignmnet guide
+view, alignmnet guide
```

Comment:
One of the other mentors corrected me that your answer of alignment guide is correct and what our materials say. Sorry for leading you down the wrong path.

Reply:
No worries!